src/main/java/it/gov/pagopa/pu/organization/service/BrokerEncryptionService.java

```diff
@@ -0,0 +1,48 @@
+package it.gov.pagopa.pu.organization.service;
+
+import it.gov.pagopa.pu.organization.dto.generated.BrokerApiKeys;
+import it.gov.pagopa.pu.organization.model.Broker;
+import it.gov.pagopa.pu.organization.util.AESUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@Slf4j
+public class BrokerEncryptionService {
+
+  private final String brokerEncryptPassword;
+
+  BrokerEncryptionService(
+    @Value("${app.brokerEncryptPassword}") String brokerEncryptPassword) {
+    this.brokerEncryptPassword = brokerEncryptPassword;
+  }
+
+  private final Map<byte[], String> apiKeyDecryptMap = Collections.synchronizedMap(new HashMap<>());
```

Comment:
let's use ConcurrentHashMap instead, it's more efficient of the synchronizedMap ;)
https://www.baeldung.com/java-synchronizedmap-vs-concurrenthashmap

```suggestion
  private final Map<byte[], String> apiKeyDecryptMap = new ConcurrentHashMap();
```

Reply:
done